Go 1.15: conversion from int to string yields a string of one rune

Go 1.15 beta 1 on Fedora Rawhide:

```
Testing    in: /builddir/build/BUILD/go-restful-3.2.0/_build/src
         PATH: /builddir/build/BUILD/go-restful-3.2.0/_build/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/sbin
       GOPATH: /builddir/build/BUILD/go-restful-3.2.0/_build:/usr/share/gocode
  GO111MODULE: off
      command: go test -buildmode pie -compiler gc -ldflags "-X github.com/emicklei/go-restful/version=3.2.0 -extldflags '-Wl,-z,relro -Wl,--as-needed  -Wl,-z,now -specs=/usr/lib/rpm/redhat/redhat-hardened-ld '"
      testing: github.com/emicklei/go-restful
github.com/emicklei/go-restful
# github.com/emicklei/go-restful
./bench_curly_test.go:17:36: conversion from int to string yields a string of one rune
./bench_curly_test.go:17:50: conversion from int to string yields a string of one rune
./bench_curly_test.go:20:37: conversion from int to string yields a string of one rune
./bench_curly_test.go:20:51: conversion from int to string yields a string of one rune
./bench_test.go:16:36: conversion from int to string yields a string of one rune
./bench_test.go:16:50: conversion from int to string yields a string of one rune
./bench_test.go:19:37: conversion from int to string yields a string of one rune
./bench_test.go:19:51: conversion from int to string yields a string of one rune
FAIL	github.com/emicklei/go-restful [build failed]
```

See https://github.com/golang/go/issues/32479

--- a/bench_curly_test.go
+++ b/bench_curly_test.go
@@ -14,10 +14,10 @@ func setupCurly(container *Container) []string {
 
 	container.Router(CurlyRouter{})
 	for i := 0; i < wsCount; i++ {
-		root := fmt.Sprintf("/%s/{%s}/", string(i+97), string(i+97))
+		root := fmt.Sprintf("/%s/{%s}/", string(rune(i+97)), string(rune(i+97)))
 		ws := new(WebService).Path(root)
 		for j := 0; j < rtCount; j++ {
-			sub := fmt.Sprintf("/%s2/{%s2}", string(j+97), string(j+97))
+			sub := fmt.Sprintf("/%s2/{%s2}", string(rune(j+97)), string(rune(j+97)))
 			ws.Route(ws.GET(sub).Consumes("application/xml").Produces("application/xml").To(echoCurly))
 		}
 		container.Add(ws)

--- a/bench_test.go
+++ b/bench_test.go
@@ -13,10 +13,10 @@ func setup(container *Container) {
 	rtCount := 26
 
 	for i := 0; i < wsCount; i++ {
-		root := fmt.Sprintf("/%s/{%s}/", string(i+97), string(i+97))
+		root := fmt.Sprintf("/%s/{%s}/", string(rune(i+97)), string(rune(i+97)))
 		ws := new(WebService).Path(root)
 		for j := 0; j < rtCount; j++ {
-			sub := fmt.Sprintf("/%s2/{%s2}", string(j+97), string(j+97))
+			sub := fmt.Sprintf("/%s2/{%s2}", string(rune(j+97)), string(rune(j+97)))
 			ws.Route(ws.GET(sub).To(echo))
 		}
 		container.Add(ws)

